### PR TITLE
implemented robot class to substitute player

### LINKED ISSUE #11: Code the BasicRobot class
A BasicRobot has (x, y) coordinates, a direction and a radius. 
It can move in the arena.

### PR DESCRIPTION
**Pull request by Phil-Ebsworth**

### CODE REVIEW

File: Robot/robot.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from pygame import Vector2
+import pygame
+
+#definition of windowsize aufgrund fehlendes game state
```
Comment (Ludwig-Work) [inline Robot/robot.py]: I believe we should try to keep all code and comments in English

File: Robot/robot.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+windowHeight = 480
+
+#robot class that defines a new player
+class Robot():
+    def __init__(self) -> None:
+        self.position = Vector2(320,240)
+        self.radius = 40
+        self.direction = Vector2(self.position.x,self.position.y + self.radius)
+        self.color = (0,0,255)
+        self.line = (self.position, Vector2(self.position,self.position.y + self.radius / 2))
```
Comment (Ludwig-Work) [inline Robot/robot.py]: I find this definition rather hard to understand. I suggest changing `direction` to a number, perhaps `direction ∊ [0, 360)` and then drawing the line based on that?
At least `direction` doesn't sound like it should be tied to points in the coordinate plane.

File: Robot/robot.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+from pygame import Vector2
+import pygame
+
+#definition of windowsize aufgrund fehlendes game state
+windowWidth = 640
+windowHeight = 480
+
+#robot class that defines a new player
+class Robot():
```
Comment (Ludwig-Work) [inline Robot/robot.py]: Class should be called `BasicRobot` according to sprint specifications.

**Review by Ludwig-Work — CHANGES_REQUESTED** [review]
I left a few review comments.
There are also a couple of typos and quite some linter warnings.

(Perhaps we should change the Flake8 CI test to treat all of them as *errors*, so they get noticed on open pull requests?)

File: Robot/robot.py (hunk 2 above)
Comment (Ludwig-Work) [reply Robot/robot.py]: `direction ∊ (-180, 180]` now, because that's how pygame's polar coordinates work.

**Review by KevinDwo — CHANGES_REQUESTED** [review]
Seems good.

1 Change:
Right now the robot can go out of bounds.
In the robot class (l. 16 + 17) we have to change the minimum 0 --> 0 + radius.

**Comment by Ludwig-Work** [comment]
Good catch! I fixed that :​)

**Review by KevinDwo — APPROVED** [review]
Good.

**Review by Ludwig-Work — APPROVED** [review]
Good for me as well now.

**Pull request merged**